src/mixins/_spacing.scss

```diff
@@ -22,7 +22,8 @@
     & > * + h3,
     & > * + h4,
     & > * + h5,
-    & > * + h6 {
+    & > * + h6,
+    & > * + [class*='heading'] {
```

Comment:
@tylersticka Are there use cases for classes other than `.c-heading`?

Reply:
I noticed that it's common for us to use class names like `{component-name}__heading`. I thought structuring the selector like this felt more indicative of what I was trying to accomplish than the specific component class name. But if others object, I could be convinced otherwise.

Reply:
@tylersticka 👍 Can you put that example in a comment?